.github/workflows/test.yml

```
@@ -32,21 +32,18 @@ jobs:
-        if: runner.os == 'macOS'
-        env:
-          HOMEBREW_NO_AUTO_UPDATE: 1
+      - &zellij-install
+        # The end-to-end tests drive `sk` inside a Zellij session (Zellij 0.44+
+        # runs on Linux, macOS and Windows). taiki-e/install-action fetches a
+        # prebuilt binary for every OS in the matrix.
+        name: Install zellij
+        uses: taiki-e/install-action@v2
+        with:
+          tool: zellij@0.44.3
+      - &locale
```

actionlint (1.7.12)

[error] anchor "locale" is defined but not used

(syntax-check)